translationCoreApps/VerseCheck
# React 16

#### This pull request addresses:
This PR adds React 16 compatibility


#### How to test this pull request:
run ```npm test```
Use component in main app and ensure usual functionality


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/translationcoreapps/versecheck/87)
